### ISSUE TITLE
Army is off the charts scaling too well with strength and doesnt not get affected by the talent [Ravenous Dead]

### ISSUE THREAD

**Issue by jonathansbb** [gamefreedomgit/Frostmourne#1322]
**Description:**
So there have been x amount of changes to DK's recently. With the latest change the talent [Ravenous Dead] no longer has an effect while army damage remains. The only difference now is that frost DKs benefit from the talent without even having it. This has been tested extensively now on the PTR. Tested with both t7 gear and bis t10 gear, with snapshots and without. The snapshot part seem to work, but as mentioned earlier the talent no longer works, its if as army is now scaling regardless if you have this or not. It is also doing too much damage.

**How to reproduce:**
So basically what needs to be done is to revert latest changes (because then it was only benefitting Unholy, as it should!) but make the scaling for the army of dead less (to reduce the damage it does i.e., make it strength less beneficial for its damage)

**How it should work:**
Unholy DK's should benefit from the talent [Ravounes Dead], meaning Unholy's army should do more damage than if specced frost.

**Database links:**
https://www.wowhead.com/wotlk/guide/classes/death-knight/unholy/dps-rotation-cooldowns-abilities-pve

**Comment by Gruonden** [comment]
There is also a disproportion in number of claws being casted in classic compared to whitemane. 670 melee hits -> 61 claws on classic. Here 400-600+ melee hits, 136 claws. Looks like number of claws is fixed, because usually you will find 136 claws even if the number of melees is lower.

https://classic.warcraftlogs.com/reports/dY13cp7byThBtLkD#fight=26&type=damage-done&source=7

**Comment by Manpower99** [comment]
Its quite obvious why the claws are being casted as "fixed" its because its not costing any energy.
Normally each claw would only be casted at 75+ energy
Here claw does not cost energy at all to begin with you can notice it by casting army and noticing their energy is always at 100.
As for the scaling, if we use classic as basis then army did not snapshot strength or benefit from the talent or glyphs at all. Blight Club discord was fully clear on this multiple times during wotlk classic. it snapshots Owner's Melee Hit(including Nerves of Cold Steel) Into Army's Melee Hit and Expertise.  It Snapshotted Owner's Stamina into a x0.2 It also snapshot's Owner's AP into a x0.065 and Melee Haste with Base Attack Speed of 2.0 it also has some base stats including agi to determine its crit chance etc.
It was very well known in wotlk classic that army's ap scaling was poor and haste scaling was very good.
Feel free to ask in the Blight Club discord if you believe otherwise.
![image](https://github.com/gamefreedomgit/Frostmourne/assets/100322493/186e5bd8-b3f5-4af0-a8f0-58e407668099)

**Comment by jonathansbb** [comment]
I see there has been put changes to the PTR. And it looks good from what I can see.

So this is on live right now:
![image](https://github.com/gamefreedomgit/Frostmourne/assets/55876132/1d7f391f-57c4-4c8a-bf87-13f69a7f9252)

this is ptr:
![image](https://github.com/gamefreedomgit/Frostmourne/assets/55876132/6b43c56f-dfc1-4d47-a9d3-c1471d236d86)

Number claws have gone from 136 to 64 and overall damage on army is also reduced. It has to be tested in full raid environment but army should be on 100-120k now as unholy if snapshotted properly, which is right where we want to be. Also consistent with classic for the potatoes who care about that. @UltraNix.

**Comment by Manpower99** [comment]
There are only small issues rn on ptr and thats the base damage being higher than it should be and the melee haste snapshot being 0.5 instead of 1:1.
But overall a massive step in the right direction.

**Comment by jonathansbb** [comment]
Ok, so now I've tested Army once again with the latest nerf on PTR and it is doing 38k snapshotted with pre-bis I hope there is coming a buff regarding the haste because right now it seems you have just nerfed it to the ground : |
